### DISCUSSION TITLE
github account

### DISCUSSION THREAD

**Discussion by Pediatra** [community/community#136235] [New to GitHub] [answered]
I've lost my 2FA methods and personal access, but I still have my password and email. Despite this, I can still commit to my repositories from my terminal. Is there a way I can use this to recover my access?

**Comment by ulyssecoispellier** [comment]
I believe that if you still have access to your account online you can deactivate 2FA and reactivate it with a separate 2FA method.

**Comment by tbrowne5** [comment, marked as answer]
Hi Pediatra,

To start, since you can still commit from the terminal, begin by making sure to clone/backup any important repositories. This will ensure you don’t lose your work in case you lose access completely.

Then, try to generate a new personal access token — If your terminal is using a cached personal access token for authentication, theres a chance it may still be valid. You can try to use git commands to check if you can still interact with github, ie:

`curl -H "Authorization: token YOUR_TOKEN" https://api.github.com/user`

There is also a 2FA recovery page that may help: [https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials](url)

If that all fails, try to contact github support for help... it's worth a shot, especially since you have the login and password.

Hopefully this information helps! Best of luck!

**Reply by ghostinhershell** [reply]
Thanks for hopping in here and offering a hand @tbrowne5! 🚀

**Reply by notebook-t** [reply]
https://github.com/user-attachments/assets/b4d4949f-c4f1-4798-9c1f-c87b264718d8

**Reply by notebook-t** [reply]
> location.foursquare.com/company/careers/attachments/assets/b4d4949f-c4f1-4798-9c1f-c87b264718d8/2024 @Pediatra

**Reply by HTony107** [reply]
![Uploading IMG_7398.png…]()

**Comment by strongmankhar** [comment]
Hello

**Reply by ghostinhershell** [reply]
Hey there @strongmankhar, 

If you have any questions, please create a new discussion in our community.

**Comment by lu6357** [comment]
Hallo

**Reply by strongmankhar** [reply]
I need help

**Comment by emmelia12345** [comment]
[Uploading Edge Mobile for Android Diagnostic Data - 20240808-2237.zip…]()

**Comment by yoloxsta** [comment]
I believe that if you still have access to your account online you can deactivate 2FA and reactivate it.

**Comment by protonsDev1** [comment]
Hi Support,
I had an account with the username protonDev, registered using the email shubham.joshi@protonshub.in.
I can no longer log into it — I’m receiving an error saying the email is not a verified personal account or is a billing-only email.

I accidentally created a new account protonDev1 but I need to regain access to protonDev as it contains repositories important to my work.

Please help me recover this account or transfer ownership of the repositories to my new account.

Thank you!

**Comment by yatin27** [comment]
Hii, support team,
My github account not working when i try to access git repository it  showing below message 
"This account is flagged, and therefore cannot authorize a third party application."
Please help ASPS.
Let me know if you need any  other details.

Github username : 
yatin27

**Comment by monty0007** [comment]
I had also the same issue, you can raise ticket on github support from your offical mail id and ask them to deactive the MFA or you can use te KEY we got in the starting.

**Comment by oudomsingz-pixel** [comment]
> ฉันสูญเสียวิธีการ 2FA และการเข้าถึงส่วนบุคคล แต่ฉันยังคงมีรหัสผ่านและอีเมลอยู่ อย่างไรก็ตาม ฉันยังคงสามารถมอบพื้นที่เก็บข้อมูลของฉันจากเทอร์มินัลของฉันได้ มีวิธีที่ฉันสามารถใช้สิ่งนี้เพื่อกู้คืนการเข้าถึงของฉันได้หรือไม่?

Touch and hold a clip to pin it. Unpinned clips will be deleted after 1 hour.
